app/assets/javascripts/application.js

```diff
@@ -21,12 +21,19 @@
 //= require_tree .
 
 function toggle(x) {
-  if (x instanceof Object) {
-    x.classList.toggle("change");
+  // if (x instanceof Object) {
```

Comment:
Is there a reason not to remove this commented out code?

Reply:
@cdoyle-temple fyi :point_up_2:

Reply:
I as thinking ahead to the refactoring (event listener) I'll be doing and wanted a history of what I'd done. It was just a quick fix to avoid a little bug.

Reply:
I'll remove it if you would prefer.

Reply:
@cdoyle-temple :man_shrugging: 
If you think it'll be cleaned up soon in another PR, then I'm fine leaving it